src/dapps/DappScreen.test.tsx

```diff
@@ -0,0 +1,718 @@
+import { fireEvent, render, within } from '@testing-library/react-native'
```

Comment:
This entire file is more or less copy-pasted from the `TabDiscover.test.tsx` file, with only minor changes to remove now-irrelevant tests and change `testID` values.